src/style.css

```diff
@@ -1,3 +1,74 @@
 body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
   margin: 0;
+  background-color: aliceblue;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+/* stylelint-disable-next-line rule-empty-line-before */
+.header {
+  background-color: #fff;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.image {
+  height: 40px;
+  width: 40px;
+  background-color: #fff;
+  margin-left: 50px;
+}
+
+.nav {
+  display: flex;
+  text-decoration: none;
+  white-space: nowrap;
+  margin-right: 50px;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.laptops {
+  color: #00acdc;
+}
+
+a {
+  margin: 10px;
+  text-decoration: none;
+  color: black;
+  font-weight: bold;
+}
+
+ul {
```

Comment:
Instead of styling all `ul`, you should add a class to the `ul` you want to style, and add the styling to that class.
The same is valid for all tags